server/auth/google/index.ts

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import passport from 'passport';
+
+const router = express.Router();
+
+router
```

Comment:
I think in other files we are not chaining multiple routes together - each get request here should be its own statement